.github/workflows/rollup-copy-test.yml

```diff
@@ -0,0 +1,45 @@
+name: Test Rollup plugin
```

Comment:
Same as with dummy-server, I am **strongly** against adding more workflows for sub packages that encode knowledge about the repo structure and build system, we have to move **away** from doing that.

Adding the _test_ script is already enough for `build.js` to automatically run the tests, no need to mess with any workflows.